src/services/formatting/smartIndenter.ts

```diff
@@ -287,6 +295,41 @@ module ts.formatting {
             }
         }
 
+        function getLineIndentationWhenExpressionIsInMultiLine(node: Node, sourceFile: SourceFile, options: EditorOptions): number {
+            // actual indentation should not be used when:
+            // - node is close parenthesis - this is the end of the expression
+            // - node is property access expression
+            if (node.kind !== SyntaxKind.CloseParenToken &&
+                node.kind !== SyntaxKind.PropertyAccessExpression &&
+                node.parent && (
+                node.parent.kind === SyntaxKind.CallExpression ||
+                node.parent.kind === SyntaxKind.NewExpression)) {
+
+                let parentExpression = (<CallExpression | NewExpression>node.parent).expression;
+                let startingExpression = getStartingExpression(<PropertyAccessExpression | CallExpression | ElementAccessExpression>parentExpression);
+
+                if (parentExpression === startingExpression) {
+                    return Value.Unknown;
+                }
+
+                let parentExpressionEnd = sourceFile.getLineAndCharacterOfPosition(parentExpression.end);
+                let startingExpressionEnd = sourceFile.getLineAndCharacterOfPosition(startingExpression.end);
+
+                if (parentExpressionEnd.line === startingExpressionEnd.line) {
+                    return Value.Unknown;
+                }
+
+                return findColumnForFirstNonWhitespaceCharacterInLine(parentExpressionEnd, sourceFile, options);
+            }
+            return Value.Unknown;
+            
+            function getStartingExpression(expression: PropertyAccessExpression | CallExpression | ElementAccessExpression) {
+                while (expression.expression)

```

Comment:
wrap loop body in curlies and add a check that `Node.kind` is equal to `PropertyAccess` / `ElementAccess` / `CallExpression`